views/funcs.html

```diff
@@ -1,36 +1,131 @@
 {{ define "header_buttons" }}
 <div id="header_buttons">
-    <a href="https://github.com/gnolang/gno"><img src="/static/img/github-mark-32px.png" alt="Github"/></a>
-</div>
-{{ end }}
+  <nav role="navigation">
+    <div id="menu-toggle">
+      <input type="checkbox" />
 
-{{ define "header_logo" }}
-  <a id="logo" href="/"><img src="/static/img/logo.png" alt="Gno.land" title="Gno.land" height="38" /></a>
-{{ end }}
+      <span></span>
```

Comment:
are these needed? if so please add some comment, otherwise delete

Reply:
@jaekwon, These `span` elements are used for burger button icon (in mobile). I also don't like empty html code and prefer pseudo-element instead, but it more convenient for this kind of icons.

Reply:
ok please add a comment.